how2meet/ui/pages/settings.py

```diff
@@ -14,7 +14,7 @@
 @router.page("/")
 def settings():
     """Settings page"""
-    frame("Settings")
```

Comment:
Seems like we can just remove settings for now

Reply:
I'm leaving it as a framework for future use.